<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_time</class>
 <widget class="QWidget" name="choose_time">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>choose_time</string>
  </property>
  <widget class="QFrame" name="LoginMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Welcome_Label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#0974ae;&quot;&gt;请选择预约时段&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close_Button">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style:outset;
background-color: rgba(255, 255, 255, 0);
border-radius:0px
}

QPushButton:hover{ 
	background-color: rgb(255, 0, 0);
}
 
QPushButton:pressed{
	background-color: rgb(220,92,102);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Hide_Button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style:outset;
background-color: rgba(255, 255, 255, 0);
border-radius:0px
}

QPushButton:hover{ 
	background-color: rgb(239, 231, 231);
}
 
QPushButton:pressed{
	background-color: rgb(243,234,234);
}</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_Button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>530</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #0076b4;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
	color: rgb(255, 255, 255);
	font: 600 10pt &quot;Segoe UI Variable Text Semibold&quot;;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}
</string>
    </property>
    <property name="text">
     <string>取消预约</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;日期：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Time_comboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 255, 250);
padding-left:10px</string>
    </property>
   </widget>
   <widget class="QLabel" name="Date_Label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;日期输出在这里&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;时段：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;时段容量：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;剩余容量：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;预约者：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;手机号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;身份证号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Visitor_ID">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>435</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;身份证号输出在这里&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Visitor_PhoneNum">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;手机号输出在这里&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Visitor_name">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;姓名输出在这里&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Visitor_comboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 255, 250);
padding-left:10px</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;人&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;人&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Available_Account">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;100&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Max_Amount">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;100&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_Button_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #0076b4;
    border: 1px solid #dcdfe6;
    padding: 10px;
    border-radius: 5px;
	color: rgb(255, 255, 255);
	font: 600 10pt &quot;Segoe UI Variable Text Semibold&quot;;
}

QPushButton:hover {
    background-color: #ecf5ff;
    color: #409eff;
}
</string>
    </property>
    <property name="text">
     <string>预约</string>
    </property>
   </widget>
   <widget class="QLabel" name="Login_Menu_Image_Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>271</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Login_Menu_Image_Label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>271</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Time_comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>choose_time</receiver>
   <slot>on_Time_combox_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>1040</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Visitor_comboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>choose_time</receiver>
   <slot>on_Person_combox_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Close_Button</sender>
   <signal>clicked()</signal>
   <receiver>choose_time</receiver>
   <slot>on_Close_Button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>941</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>1039</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Hide_Button</sender>
   <signal>clicked()</signal>
   <receiver>choose_time</receiver>
   <slot>on_Hide_Button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>902</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>1049</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Exit_Button</sender>
   <signal>clicked()</signal>
   <receiver>choose_time</receiver>
   <slot>on_Close_Button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>1014</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Exit_Button_2</sender>
   <signal>clicked()</signal>
   <receiver>choose_time</receiver>
   <slot>on_Appoint_Button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_Time_combox_changed()</slot>
  <slot>on_Person_combox_changed()</slot>
  <slot>on_Close_Button_clicked()</slot>
  <slot>on_Hide_Button_clicked()</slot>
  <slot>on_Appoint_Button_clicked()</slot>
 </slots>
</ui>
